snapcraft/internal/errors.py

```diff
@@ -246,6 +246,11 @@ def __init__(self, message):
         super().__init__(message=message)
 
 
+class PluginCommandFailure(SnapcraftError):
+
+    fmt = 'Build failed'
```

Comment:
I understand not wanting to print a traceback in this case (which is a debatable in the first place), but this would be the most useless error message known to man :stuck_out_tongue: . Can we make it at least something like: `Build failed: 'my command that failed' exited with an error code of 1`?

Reply:
Your reply makes me think you've never seen what a failure in that code path looks like. You'll get the verbatim console output including the original command. Adding that to the exception would be rather redundant.

Reply:
Okay, so what is the original command that has failed in this case?

```
Building Catkin packages...
/bin/bash /tmp/tmpto1fx5ay
Base path: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build
Source space: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src
Build space: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/build_isolated
Devel space: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/devel_isolated
Install space: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/opt/ros/kinetic
Additional CMake Arguments: -DCMAKE_C_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu  -DCMAKE_CXX_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu/c++/5.4.0 -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/c++/5.4.0 -DCMAKE_LD_FLAGS= -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib/x86_64-linux-gnu -DCMAKE_C_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/gcc -DCMAKE_CXX_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/g++ -DCMAKE_BUILD_TYPE=Release
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~  traversing 2 packages in topological order:
~~  - listener
~~  - talker
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The packages or cmake arguments have changed, forcing cmake invocation

==> Processing catkin package: 'listener'
==> Creating build directory: 'build_isolated/listener'
==> cmake /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src/listener -DCATKIN_DEVEL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/devel_isolated/listener -DCMAKE_INSTALL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/opt/ros/kinetic -DCMAKE_C_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu  -DCMAKE_CXX_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu/c++/5.4.0 -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/c++/5.4.0 -DCMAKE_LD_FLAGS= -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib/x86_64-linux-gnu -DCMAKE_C_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/gcc -DCMAKE_CXX_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/g++ -DCMAKE_BUILD_TYPE=Release -G Unix Makefiles in '/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/build_isolated/listener'
CMake Error: Error in cmake code at
/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src/listener/CMakeLists.txt:19:
Parse error.  Function missing ending ")".  End of file reached.
-- Configuring incomplete, errors occurred!
<== Failed to process package 'listener': 
  Command '['cmake', '/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src/listener', '-DCATKIN_DEVEL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/devel_isolated/listener', '-DCMAKE_INSTALL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/opt/ros/kinetic', '-DCMAKE_C_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu ', '-DCMAKE_CXX_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu/c++/5.4.0 -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/c++/5.4.0', '-DCMAKE_LD_FLAGS= -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib/x86_64-linux-gnu', '-DCMAKE_C_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/gcc', '-DCMAKE_CXX_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/g++', '-DCMAKE_BUILD_TYPE=Release', '-G', 'Unix Makefiles']' returned non-zero exit status 1

Reproduce this error by running:
==> cd /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/build_isolated/listener && cmake /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src/listener -DCATKIN_DEVEL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/devel_isolated/listener -DCMAKE_INSTALL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/opt/ros/kinetic '-DCMAKE_C_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu ' '-DCMAKE_CXX_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu/c++/5.4.0 -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/c++/5.4.0' '-DCMAKE_LD_FLAGS= -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib/x86_64-linux-gnu' -DCMAKE_C_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/gcc -DCMAKE_CXX_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/g++ -DCMAKE_BUILD_TYPE=Release -G 'Unix Makefiles'

Command failed, exiting.
Build failed
```

Reply:
Okay, so what is the original command that has failed in this case?

```
$ snapcraft
"grade" property not specified: defaulting to "stable"
Preparing to pull ros-project 
Get:1 http://packages.ros.org/ros/ubuntu xenial InRelease [4037 B]                                    
Ign http://packages.ros.org/ros/ubuntu xenial InRelease                                               
Get:2 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]                            
Hit http://archive.ubuntu.com/ubuntu xenial InRelease                                                 
Get:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]                              
Get:4 http://archive.ubuntu.com/ubuntu xenial-security InRelease [102 kB]                             
Fetched 310 kB in 0s (0 B/s)                                                                          
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgssapi3-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-exception1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-python1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-six_1.10.0-3_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-wave-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libmpx0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-atomic1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdistro_0.6.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/init-system-helpers_1.29ubuntu4_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libltdl-dev_2.4.6-0.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-math-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-test1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rospkg-modules_1.1.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libkrb5-3_1.13.2+dfsg-5ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost1.58-tools-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgssapi-krb5-2_1.13.2+dfsg-5ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python2.7-dev_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgcc-5-dev_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-log1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-program-options1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libstdc++-4.8-dev_4.8.5-4ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-locale1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-catkin-pkg-modules_0.3.6-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libwind0-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph-parallel1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-chrono-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-iostreams-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsasl2-modules-db_2.1.26.dfsg1-14build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-thread1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/linux-libc-dev_4.4.0-93.116_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libcurl3_7.47.0-1ubuntu2.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhwloc5_1.11.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libldap-2.4-2_2.4.42+dfsg-2ubuntu3.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roslib_1.13.5-0xenial-20170308-181854-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/sgml-base_1.26+nmu4ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/openssl_1.0.2g-1ubuntu4.8_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/docutils-common_0.12+dfsg-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph-parallel1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libibverbs1_1.1.8-1.1ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libc6-dev_2.23-0ubuntu9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-filesystem1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-yaml_3.11-3build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-timer1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-wave1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-thread-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdep_0.11.8-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-coroutine1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-empy_3.3.2-1build1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-math1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libexpat1-dev_2.1.0-7ubuntu0.16.04.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libheimntlm0-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-system-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/gcc-4.8-base_4.8.5-4ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python2.7-minimal_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-all-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-context1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsqlite3-0_3.11.0-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libicu55_55.1-7ubuntu0.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/binutils_2.26.1-1ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-date-time1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libarchive13_3.1.2-11ubuntu0.16.04.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libitm1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-thread1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgmp10_6.1.0+dfsg-2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-log-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-serialization1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/perl-modules-5.22_5.22.1-9_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python2.7_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-catkin-pkg_0.3.6-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-locale1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libheimbase1-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-atomic1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libnuma1_2.0.11-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-system1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3.5_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-python1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3.5-minimal_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhwloc-dev_1.11.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-random1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libk5crypto3_1.13.2+dfsg-5ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-random-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7-dev_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-date-time1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libcilkrts5_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/mime-support_3.59ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/cmake-data_3.5.1-1ubuntu3_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsasl2-2_2.1.26.dfsg1-14build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libasan0_4.8.5-4ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libasan2_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-docutils_0.12+dfsg-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/openmpi-common_1.10.2-8ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libquadmath0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-catkin_0.7.6-0xenial-20170217-212821-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/gcc-6-base_6.0.1-0ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libmpdec2_2.4.2-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-timer-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libprocps4_3.3.10-4ubuntu2.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-serialization-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libexpat1_2.1.0-7ubuntu0.16.04.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rospkg_1.1.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libltdl7_2.4.6-0.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libperl5.22_5.22.1-9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-signals-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython-stdlib_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/mpi-default-bin_1.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/xml-core_0.13+nmu2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-regex1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libstdc++-5-dev_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3-minimal_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-serialization1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgcc-4.8-dev_4.8.5-4ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/librtmp1_2.4+20151223.gitfa8646d-1ubuntu0.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-program-options-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libubsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/mpi-default-dev_1.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-filesystem1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi-python-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph-parallel-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/pkg-config_0.29.1-0ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libkrb5support0_1.13.2+dfsg-5ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython-dev_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3-stdlib_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/make_4.1-6_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-roman_2.0.0-2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libopenmpi1.10_1.10.2-8ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-context1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-iostreams1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgdbm3_1.8.3-13.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhcrypto4-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-pkg-resources_20.7.0-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libkeyutils1_1.5.9-8ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-nose_1.3.7-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/distro-info-data_0.28ubuntu0.3_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/xz-utils_5.1.1alpha+20120614-2ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-wave1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgomp1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/dh-python_2.20151103ubuntu1.1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-exception-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-locale-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-dev_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhogweed4_3.2-1ubuntu0.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtinyxml2.6.2v5_2.6.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-setuptools_20.7.0-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-test1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-log1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/bzip2_1.0.6-8_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rospack_2.3.3-0xenial-20170308-181000-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-date-time-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libnettle6_3.2-1ubuntu0.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-signals1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/lsb-release_9.20160110ubuntu0.2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libc-dev-bin_2.23-0ubuntu9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libxml2_2.9.3+dfsg1-1ubuntu0.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libidn11_1.32-3ubuntu1.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-python-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/cmake_3.5.1-1ubuntu3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7-minimal_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtinyxml-dev_2.6.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgnutls30_3.4.10-4ubuntu1.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libroken18-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ca-certificates_20160104ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-math1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libicu-dev_55.1-7ubuntu0.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/perl_5.22.1-9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libffi6_3.2.1-4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-coroutine1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libp11-kit0_0.23.2-5~ubuntu16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libasn1-8-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-atomic-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libkrb5-26-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3.5-stdlib_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-regex-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-filesystem-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libatomic1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libjsoncpp1_1.7.2-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtasn1-6_4.7-3ubuntu0.16.04.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-iostreams1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/icu-devtools_55.1-7ubuntu0.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-dateutil_2.4.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libyaml-0-2_0.1.6-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-chrono1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi-python1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7-stdlib_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/openmpi-bin_1.10.2-8ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-signals1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3.5-minimal_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libdpkg-perl_1.18.4ubuntu1.2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhx509-5-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/patch_2.7.5-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libibverbs-dev_1.1.8-1.1ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-regex1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-minimal_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdistro-modules_0.6.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libglib2.0-0_2.48.2-0ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libopenmpi-dev_1.10.2-8ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-tools-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi-python1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgtest-dev_1.7.0-4ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-system1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-timer1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-coroutine-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-context-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-random1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-chrono1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libssl1.0.0_1.0.2g-1ubuntu4.8_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-test-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/dpkg-dev_1.18.4ubuntu1.2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libnuma-dev_2.0.11-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblzo2-2_2.08-1.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-program-options1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Copying needed target link from the system /lib/x86_64-linux-gnu/libgcc_s.so.1
Pulling ros-project 
Preparing to fetch compilers...
Fetching compilers...
Get:1 http://packages.ros.org/ros/ubuntu xenial InRelease [4037 B]                                    
Ign http://packages.ros.org/ros/ubuntu xenial InRelease                                               
Hit http://archive.ubuntu.com/ubuntu xenial InRelease                                                 
Get:2 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]                            
Get:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]                              
Get:4 http://archive.ubuntu.com/ubuntu xenial-security InRelease [102 kB]                             
Fetched 310 kB in 0s (0 B/s)                                                                          
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libmpc3_1.0.3-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libmpx0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgcc-5-dev_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/linux-libc-dev_4.4.0-93.116_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libmpfr4_3.1.4-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libc6-dev_2.23-0ubuntu9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/cpp-5_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/binutils_2.26.1-1ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/cpp_5.3.1-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libitm1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgmp10_6.1.0+dfsg-2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/g++_5.3.1-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/gcc_5.3.1-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libcilkrts5_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libasan2_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libquadmath0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/gcc-6-base_6.0.1-0ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libisl15_0.16.1-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libstdc++-5-dev_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libubsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/g++-5_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/gcc-5_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgomp1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libc-dev-bin_2.23-0ubuntu9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libatomic1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libcc1-0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Installing compilers...
Copying needed target link from the system /lib/x86_64-linux-gnu/libgcc_s.so.1
Preparing to fetch rosdep...
Fetching rosdep...
Get:1 http://packages.ros.org/ros/ubuntu xenial InRelease [4037 B]                                    
Ign http://packages.ros.org/ros/ubuntu xenial InRelease                                               
Hit http://archive.ubuntu.com/ubuntu xenial InRelease                                                 
Get:2 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]                            
Get:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]                              
Get:4 http://archive.ubuntu.com/ubuntu xenial-security InRelease [102 kB]                             
Fetched 310 kB in 0s (0 B/s)                                                                          
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-six_1.10.0-3_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdistro_0.6.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rospkg-modules_1.1.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-catkin-pkg-modules_0.3.6-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/sgml-base_1.26+nmu4ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/openssl_1.0.2g-1ubuntu4.8_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/docutils-common_0.12+dfsg-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-yaml_3.11-3build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdep_0.11.8-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python2.7-minimal_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsqlite3-0_3.11.0-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python2.7_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-catkin-pkg_0.3.6-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3.5_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3.5-minimal_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/mime-support_3.59ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-docutils_0.12+dfsg-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/gcc-6-base_6.0.1-0ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libmpdec2_2.4.2-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libexpat1_2.1.0-7ubuntu0.16.04.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rospkg_1.1.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython-stdlib_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/xml-core_0.13+nmu2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3-minimal_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3-stdlib_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-roman_2.0.0-2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-pkg-resources_20.7.0-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/distro-info-data_0.28ubuntu0.3_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/dh-python_2.20151103ubuntu1.1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-setuptools_20.7.0-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/lsb-release_9.20160110ubuntu0.2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7-minimal_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ca-certificates_20160104ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libffi6_3.2.1-4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3.5-stdlib_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-dateutil_2.4.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libyaml-0-2_0.1.6-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7-stdlib_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3.5-minimal_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-minimal_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdistro-modules_0.6.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libssl1.0.0_1.0.2g-1ubuntu4.8_amd64.deb
Installing rosdep...
Initializing rosdep database...
Updating rosdep database...
Determining system dependencies for Catkin packages...
Preparing to fetch package dependencies...
Fetching package dependencies...
Get:1 http://packages.ros.org/ros/ubuntu xenial InRelease [4037 B]                                    
Ign http://packages.ros.org/ros/ubuntu xenial InRelease                                               
Get:2 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]                            
Hit http://archive.ubuntu.com/ubuntu xenial InRelease                                                 
Get:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]                              
Get:4 http://archive.ubuntu.com/ubuntu xenial-security InRelease [102 kB]                             
Fetched 310 kB in 0s (0 B/s)                                                                          
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgssapi3-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-exception1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-python1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-six_1.10.0-3_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-wave-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libmpx0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosparam_1.12.7-0xenial-20170217-215616-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-atomic1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdistro_0.6.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/init-system-helpers_1.29ubuntu4_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libltdl-dev_2.4.6-0.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-math-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-test1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rospkg-modules_1.1.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rospy_1.12.7-0xenial-20170613-172037-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libkrb5-3_1.13.2+dfsg-5ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roslaunch_1.12.7-0xenial-20170613-174031-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost1.58-tools-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roscpp-traits_0.6.2-0xenial-20170217-215757-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgssapi-krb5-2_1.13.2+dfsg-5ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python2.7-dev_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgcc-5-dev_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-log1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblog4cxx10v5_0.10.0-10ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-program-options1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libstdc++-4.8-dev_4.8.5-4ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblapack3_3.6.0-2ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsctp1_1.0.16+dfsg-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-locale1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roslisp_1.9.21-0xenial-20170714-101403-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libldap2-dev_2.4.42+dfsg-2ubuntu3.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-catkin-pkg-modules_0.3.6-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-ros-comm_1.12.7-0xenial-20170714-101801-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libwind0-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph-parallel1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-ros-core_1.3.1-0xenial-20170714-102336-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-chrono-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-iostreams-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsasl2-modules-db_2.1.26.dfsg1-14build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-thread1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/linux-libc-dev_4.4.0-93.116_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libcurl3_7.47.0-1ubuntu2.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhwloc5_1.11.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libldap-2.4-2_2.4.42+dfsg-2ubuntu3.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-topic-tools_1.12.7-0xenial-20170613-182844-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roslib_1.13.5-0xenial-20170308-181854-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-diagnostic-msgs_1.12.5-0xenial-20170613-161627-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosgraph-msgs_1.11.2-0xenial-20170613-161833-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/sgml-base_1.26+nmu4ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/openssl_1.0.2g-1ubuntu4.8_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roslz4_1.12.7-0xenial-20170308-182349-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/docutils-common_0.12+dfsg-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph-parallel1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libibverbs1_1.1.8-1.1ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libc6-dev_2.23-0ubuntu9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosmsg_1.12.7-0xenial-20170613-183603-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-filesystem1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-yaml_3.11-3build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-cmake-modules_0.4.1-0xenial-20170227-125155-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosservice_1.12.7-0xenial-20170613-183834-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-timer1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-wave1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libaprutil1_1.5.4-1build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-thread-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-ros_1.13.5-0xenial-20170613-170133-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdep_0.11.8-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-coroutine1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-empy_3.3.2-1build1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-math1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libexpat1-dev_2.1.0-7ubuntu0.16.04.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libheimntlm0-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosout_1.12.7-0xenial-20170613-173704-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-system-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-cpp-common_0.6.2-0xenial-20170217-214958-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/gcc-4.8-base_4.8.5-4ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-trajectory-msgs_1.12.5-0xenial-20170613-163033-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python2.7-minimal_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-all-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-context1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsqlite3-0_3.11.0-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libicu55_55.1-7ubuntu0.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/binutils_2.26.1-1ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-netifaces_0.10.4-0.1build2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roscpp-serialization_0.6.2-0xenial-20170217-215905-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-date-time1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libarchive13_3.1.2-11ubuntu0.16.04.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libitm1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-nav-msgs_1.12.5-0xenial-20170613-162132-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-thread1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-stereo-msgs_1.12.5-0xenial-20170613-162914-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgmp10_6.1.0+dfsg-2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-log-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-serialization1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/perl-modules-5.22_5.22.1-9_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-visualization-msgs_1.12.5-0xenial-20170613-163040-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python2.7_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-catkin-pkg_0.3.6-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-locale1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libheimbase1-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-atomic1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosclean_1.13.5-0xenial-20170217-215645-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libnuma1_2.0.11-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-system1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3.5_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-gennodejs_2.0.1-0xenial-20170217-220428-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosbuild_1.13.5-0xenial-20170613-153404-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-python1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3.5-minimal_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhwloc-dev_1.11.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblog4cxx-dev_0.10.0-10ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-random1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libk5crypto3_1.13.2+dfsg-5ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-random-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7-dev_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-date-time1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libcilkrts5_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/mime-support_3.59ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/cmake-data_3.5.1-1ubuntu3_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsasl2-2_2.1.26.dfsg1-14build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libasan0_4.8.5-4ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libasan2_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libbz2-dev_1.0.6-8_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-docutils_0.12+dfsg-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/openmpi-common_1.10.2-8ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libquadmath0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roswtf_1.12.7-0xenial-20170613-195213-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roscreate_1.13.5-0xenial-20170217-215350-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-catkin_0.7.6-0xenial-20170217-212821-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/gcc-6-base_6.0.1-0ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libmpdec2_2.4.2-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-timer-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libprocps4_3.3.10-4ubuntu2.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-serialization-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libexpat1_2.1.0-7ubuntu0.16.04.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosgraph_1.12.7-0xenial-20170217-215422-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosbag-storage_1.12.7-0xenial-20170308-191526-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rospkg_1.1.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libltdl7_2.4.6-0.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libsctp-dev_1.0.16+dfsg-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libperl5.22_5.22.1-9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libapr1_1.5.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-crypto_2.6.1-6ubuntu0.16.04.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-signals-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython-stdlib_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/mpi-default-bin_1.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/xml-core_0.13+nmu2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-regex1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-genpy_0.6.5-0xenial-20170308-181016-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libstdc++-5-dev_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python3-minimal_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-serialization1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-gencpp_0.5.5-0xenial-20170308-180846-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgcc-4.8-dev_4.8.5-4ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/librtmp1_2.4+20151223.gitfa8646d-1ubuntu0.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-program-options-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libubsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/mpi-default-dev_1.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-filesystem1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi-python-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph-parallel-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-message-filters_1.12.7-0xenial-20170613-183116-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-paramiko_1.16.0-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libblas-common_3.6.0-2ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/pkg-config_0.29.1-0ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libkrb5support0_1.13.2+dfsg-5ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-mk_1.13.5-0xenial-20170613-165407-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-geneus_2.2.6-0xenial-20170613-152311-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython-dev_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-message-runtime_0.4.12-0xenial-20170308-181648-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3-stdlib_3.5.1-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosunit_1.13.5-0xenial-20170308-182205-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-sensor-msgs_1.12.5-0xenial-20170613-162039-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgfortran3_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/make_4.1-6_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-roman_2.0.0-2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libopenmpi1.10_1.10.2-8ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-context1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-ecdsa_0.13-2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-iostreams1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgdbm3_1.8.3-13.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosmake_1.13.5-0xenial-20170217-215441-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhcrypto4-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosnode_1.12.7-0xenial-20170613-194831-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-genlisp_0.4.16-0xenial-20170217-220601-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-pkg-resources_20.7.0-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libkeyutils1_1.5.9-8ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-defusedxml_0.4.1-2ubuntu0.16.04.1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblz4-1_0.0~r131-2ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-nose_1.3.7-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/distro-info-data_0.28ubuntu0.3_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/xz-utils_5.1.1alpha+20120614-2ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-wave1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosmaster_1.12.7-0xenial-20170217-215722-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblsan0_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgomp1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/dh-python_2.20151103ubuntu1.1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rostest_1.12.7-0xenial-20170613-182517-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-exception-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libaprutil1-dev_1.5.4-1build1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-std-msgs_0.5.11-0xenial-20170613-160441-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-locale-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-dev_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/uuid-dev_2.27.1-6ubuntu3.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhogweed4_3.2-1ubuntu0.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-geometry-msgs_1.12.5-0xenial-20170613-161430-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtinyxml2.6.2v5_2.6.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-setuptools_20.7.0-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-test1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-log1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosconsole_1.12.7-0xenial-20170613-165454-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosconsole-bridge_0.4.4-0xenial-20170613-174724-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/bzip2_1.0.6-8_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rospack_2.3.3-0xenial-20170308-181000-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-genmsg_0.5.8-0xenial-20170217-220248-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-date-time-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libapr1-dev_1.5.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libnettle6_3.2-1ubuntu0.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rostime_0.6.2-0xenial-20170217-215458-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-signals1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/lsb-release_9.20160110ubuntu0.2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libc-dev-bin_2.23-0ubuntu9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libxml2_2.9.3+dfsg1-1ubuntu0.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-graph1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosbash_1.13.5-0xenial-20170217-215234-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-shape-msgs_1.12.5-0xenial-20170613-162943-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libidn11_1.32-3ubuntu1.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-python-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/cmake_3.5.1-1ubuntu3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7-minimal_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtinyxml-dev_2.6.2-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgnutls30_3.4.10-4ubuntu1.3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libroken18-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ca-certificates_20160104ubuntu1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-math1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libicu-dev_55.1-7ubuntu0.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/perl_5.22.1-9_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libffi6_3.2.1-4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-coroutine1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libp11-kit0_0.23.2-5~ubuntu16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libasn1-8-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-atomic-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosbag-migration-rule_1.0.0-0xenial-20170217-225238-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libkrb5-26-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-message-generation_0.4.0-0xenial-20170613-152756-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-xmlrpcpp_1.12.7-0xenial-20170217-215645-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblog4cxx10-dev_0.10.0-10ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3.5-stdlib_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-regex-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-filesystem-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libatomic1_5.4.0-6ubuntu1~16.04.4_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libjsoncpp1_1.7.2-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libconsole-bridge-dev_0.3.2-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-common-msgs_1.12.5-0xenial-20170613-172402-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-actionlib-msgs_1.12.5-0xenial-20170613-160720-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libtasn1-6_4.7-3ubuntu0.16.04.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-iostreams1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libblas3_3.6.0-2ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/icu-devtools_55.1-7ubuntu0.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/sbcl_1.3.1-1ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-dateutil_2.4.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libyaml-0-2_0.1.6-3_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rostopic_1.12.7-0xenial-20170613-183627-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-chrono1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi-python1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libconsole-bridge0.2v5_0.3.2-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython2.7-stdlib_2.7.12-1ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/openmpi-bin_1.10.2-8ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roslang_1.13.5-0xenial-20170217-220418-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-signals1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libpython3.5-minimal_3.5.2-2ubuntu0~16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosbag_1.12.7-0xenial-20170613-183226-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libdpkg-perl_1.18.4ubuntu1.2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libhx509-5-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/patch_2.7.5-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libibverbs-dev_1.1.8-1.1ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-regex1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roscpp_1.12.7-0xenial-20170613-165746-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-numpy_1.11.0-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-minimal_2.7.11-1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/python-rosdistro-modules_0.6.2-1_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libglib2.0-0_2.48.2-0ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libopenmpi-dev_1.10.2-8ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-std-srvs_1.11.2-0xenial-20170613-160502-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-tools-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblz4-dev_0.0~r131-2ubuntu2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-mpi-python1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-roscpp-core_0.6.2-0xenial-20170217-220156-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libgtest-dev_1.7.0-4ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-system1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/ros-kinetic-rosboost-cfg_1.13.5-0xenial-20170217-215147-0800_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-timer1.58.0_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-coroutine-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-context-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-random1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-chrono1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libssl1.0.0_1.0.2g-1ubuntu4.8_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-test-dev_1.58.0.1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/dpkg-dev_1.18.4ubuntu1.2_all.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libnuma-dev_2.0.11-1ubuntu1_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/liblzo2-2_2.08-1.2_amd64.deb
Ignoring already existing file: /home/kyrofa/.cache/snapcraft/stage-packages/apt/a8b5df7d94c8ee9724b629862f2128bc14deaae5b5fb7745a756221f861016bbd93c6f47e4c5dff59dc12ecdbda02273/var/cache/apt/archives/libboost-program-options1.58-dev_1.58.0+dfsg-5ubuntu3.1_amd64.deb
Installing package dependencies...
Copying needed target link from the system /lib/x86_64-linux-gnu/libuuid.so.1.3.0
Copying needed target link from the system /lib/x86_64-linux-gnu/libbz2.so.1.0.4
Preparing to build ros-project 
Building ros-project 
Preparing to build Catkin packages...
Building Catkin packages...
/bin/bash /tmp/tmpto1fx5ay
Base path: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build
Source space: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src
Build space: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/build_isolated
Devel space: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/devel_isolated
Install space: /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/opt/ros/kinetic
Additional CMake Arguments: -DCMAKE_C_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu  -DCMAKE_CXX_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu/c++/5.4.0 -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/c++/5.4.0 -DCMAKE_LD_FLAGS= -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib/x86_64-linux-gnu -DCMAKE_C_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/gcc -DCMAKE_CXX_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/g++ -DCMAKE_BUILD_TYPE=Release
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~  traversing 2 packages in topological order:
~~  - listener
~~  - talker
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The packages or cmake arguments have changed, forcing cmake invocation

==> Processing catkin package: 'listener'
==> Creating build directory: 'build_isolated/listener'
==> cmake /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src/listener -DCATKIN_DEVEL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/devel_isolated/listener -DCMAKE_INSTALL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/opt/ros/kinetic -DCMAKE_C_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu  -DCMAKE_CXX_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu/c++/5.4.0 -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/c++/5.4.0 -DCMAKE_LD_FLAGS= -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib/x86_64-linux-gnu -DCMAKE_C_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/gcc -DCMAKE_CXX_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/g++ -DCMAKE_BUILD_TYPE=Release -G Unix Makefiles in '/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/build_isolated/listener'
CMake Error: Error in cmake code at
/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src/listener/CMakeLists.txt:19:
Parse error.  Function missing ending ")".  End of file reached.
-- Configuring incomplete, errors occurred!
<== Failed to process package 'listener': 
  Command '['cmake', '/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src/listener', '-DCATKIN_DEVEL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/devel_isolated/listener', '-DCMAKE_INSTALL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/opt/ros/kinetic', '-DCMAKE_C_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu ', '-DCMAKE_CXX_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu/c++/5.4.0 -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/c++/5.4.0', '-DCMAKE_LD_FLAGS= -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib/x86_64-linux-gnu', '-DCMAKE_C_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/gcc', '-DCMAKE_CXX_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/g++', '-DCMAKE_BUILD_TYPE=Release', '-G', 'Unix Makefiles']' returned non-zero exit status 1

Reproduce this error by running:
==> cd /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/build_isolated/listener && cmake /home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/src/listener -DCATKIN_DEVEL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/build/devel_isolated/listener -DCMAKE_INSTALL_PREFIX=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/opt/ros/kinetic '-DCMAKE_C_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu ' '-DCMAKE_CXX_FLAGS= -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu/c++/5.4.0 -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/x86_64-linux-gnu -I/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/include/c++/5.4.0' '-DCMAKE_LD_FLAGS= -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/install/usr/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/lib/x86_64-linux-gnu -L/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/lib/x86_64-linux-gnu' -DCMAKE_C_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/gcc -DCMAKE_CXX_COMPILER=/home/kyrofa/src/snapcraft/demos/ros/parts/ros-project/compilers/install/usr/bin/g++ -DCMAKE_BUILD_TYPE=Release -G 'Unix Makefiles'

Command failed, exiting.
Build failed
```

This is not the best example, but my point is that such information is easy to lose, particularly when the build process is verbose. When you exit with something equivalent to "oops," you're not being as helpful as you could be.

Reply:
I'd say the `Ignoring already existing file` spam is horrible, but the relevant error is very easy to spot in the last lines. This is why it seemed redundant to me to included in the exception.

But I see your point. I happen to be used to CMake. It might not be easy to spot.

Reply:
Indeed. The command run is _not_ cmake.